docs/source/configuration/key_value_service_configs/oracle_key_value_service_config.rst

```diff
@@ -17,9 +17,9 @@ runtime dependency. In gradle this looks like:
 
 e.g.
 
-.. code-block:: groovy
+.. code-block-with-version-replacement:: groovy
 
-  runtime 'com.palantir.atlasdb:atlasdb-dbkvs:0.19.0'
+  runtime 'com.palantir.atlasdb:atlasdb-dbkvs:|version|'
```

Comment:
I'm pretty sure it is: I grep'd for "develop", "0.[0-9].0",  and "0.[0-9]+", and this is all I could find.